draft-ietf-rats-architecture.md

```diff
@@ -106,8 +106,8 @@ such as roles, device composition, topological models, and appraisal is vital fo
 semantic interoperability across solutions and platforms involving multiple vendors and providers.
 
 Amongst other things, this document is about trust and trustworthiness.
-Trust is a decision being made. Trustworthiness is a quality that is
-assessed via evidence created. This is a subtle difference and being
+To trust is a decision that spans beyond the technical assertions. Trustworthiness is a quality that is
```

Comment:
The decision to trust spans beyond technical assertions to focus on trustworthiness as a quality of the computing environment.

Reply:
> The decision to trust spans beyond technical assertions to focus on trustworthiness as a quality of the computing environment.

I'm unclear if you are agreeing or disagreeing here.

Reply:
@mcr: I think this is the replacement text that Ned suggests to use.